scripts/update.py

```diff
@@ -47,11 +48,21 @@
     "blast": {
         "mainnet": 81457,
         "sepolia": 168587773,
+        "testnet": 23888,
     },
     "bsc": {
         "mainnet": 56,
         "testnet": 97,
         "opBNB": 204,
+        "opBNB-testnet": 5611,
```

Comment:
question: Are we sure we want these to be capitalized? I assume everything leading up to this point is case sensitive.

Reply:
ya lower is probably better

Reply:
done!